### ISSUE TITLE
How to add urdu fasttext vec to rasa with spacy backend????

### ISSUE THREAD

**Issue by Talhag958** [RasaHQ/rasa#5013]

**Comment by JulianGerhard21** [comment]
Hi @Talhag958 ,

I am not quite sure what you want to achieve - I assume you want to build a bot for the Urdu language. Training a model in spacy using custom word vectors like the fasttext ones is pretty easy and can be seen [here](https://spacy.io/usage/vectors-similarity#converting). 

Please note that you need to start with a **blank** model since there is no pretrained one for Urdu. If you plan to use this model for different downstream tasks, you maybe need a good amount of training data despite the fasttext embeddings, e.g. for extracting named entities.

Keep in mind, that Urdu has some interesting / different **punctuation** symbols so you need to take care of the **tokenizer** aswell, depending on your pipeline.

Regards
Julian

**Comment by Talhag958** [comment]
Julian, thanks very much. I have found this blog [build Urdu language model in SpaCy](https://www.urdunlp.com/2019/08/how-to-build-urdu-language-model-in.html). How much i furter need to train this model for rasa........

**Comment by sara-tagger** [comment]
Thanks for the question, our Github issue board is for **bugs and feature requests only,** though 😕
Please ask this question in the [forum✨](https://forum.rasa.com/c/rasa)

###### If you were describing a bug, please create a [report](https://github.com/RasaHQ/rasa/issues/new?labels=bug&template=bug_report.md) and fill out the template.
